geyser-plugin-manager/src/geyser_plugin_manager.rs

```diff
@@ -72,4 +52,446 @@ impl GeyserPluginManager {
         }
         false
     }
+
+    /// Admin RPC request handler
+    pub(crate) fn list_plugins(&self) -> JsonRpcResult<Vec<String>> {
+        Ok(self.plugins.iter().map(|p| p.name().to_owned()).collect())
+    }
+
+    /// Admin RPC request handler
+    /// # Safety
+    ///
+    /// This function loads the dynamically linked library specified in the path. The library
+    /// must do necessary initializations.
+    pub(crate) fn load_plugin(
+        &mut self,
+        geyser_plugin_config_file: impl AsRef<Path>,
+    ) -> JsonRpcResult<String> {
+        // First load plugin
+        let (mut new_plugin, new_lib, new_config_file) =
+            load_plugin_from_config(geyser_plugin_config_file.as_ref()).map_err(|e| {
+                jsonrpc_core::Error {
+                    code: ErrorCode::InternalError,
```

Comment:
I think most errors returning from that function can be classified as InvalidRequest

Reply:
done. will push